roles/ldap_kerberos/defaults/main.yml

```diff
@@ -86,4 +86,33 @@ kerberos_kadmin_admin_template: "*/admin@{{ realm }}"
 kerberos_renew_lifetime: "7d 0h 0m 0s"
 kerberos_max_renewable_life: "7d 0h 0m 0s"
 
-kerberos_server: "{{ groups['kdc'][0] }}.{{ domain }}"
+kerberos_servers: >-
+  {{
+    groups['kdc']
+    | map('regex_replace', '^(.*)$', '\1.' + domain)
+    | list
+  }}
+
+kdcproxy_servers: >-
+  {{
+    groups['kdcproxy']
+    | map('regex_replace', '^(.*)$', 'https://\1.' + domain + '/KdcProxy')
+    | list
+  }}
+
+certificates_path: /etc/ssl/certs
+certificates_ca_filename: tdp_ca
+
+httpd_ca_root: "{{ certificates_path }}/{{ certificates_ca_filename }}.crt"
```

Comment:
Is this variable used at some point?

Reply:
yes

Reply:
In ssl-httpd.conf.j2, it's templated inside

Reply:
Ok, it was missing, I put it back